wgpu/backends/rs.py

```diff
@@ -90,13 +90,17 @@ def _get_wgpu_lib_path():
     if override_path:
         return override_path
 
+    # Load the debug binary if requested
+    debug_mode = bool(os.getenv("WGPU_DEBUG", "").strip())
```

Comment:
Setting WGPU_DEBUG to "0" or "false" will still resolve to true. Should we be more specific here?

Reply:
Fair enough. I'll specifically test for "1"